src/Libraries/RevitNodes/Elements/Revision.cs

```diff
@@ -167,12 +167,28 @@ private static ElementId GetRevisionNumberingSequence(RevisionNumberType numberT
             foreach (var id in elementids)
             {
                 var seqElem = doc.GetElement(id) as RevisionNumberingSequence;
-                if (seqElem.NumberType == numberType)
+                if (seqElem.Name.StartsWith("Dynamo_") && seqElem.NumberType == numberType)
                 {
                     revisionNumberingSequenceId = seqElem.Id;
                     break;
                 }
             }
+            if(revisionNumberingSequenceId == ElementId.InvalidElementId)
+            {
+                RevisionNumberingSequence revisionNumberingSequence;
+                switch (numberType)
+                {
+                    case RevisionNumberType.Alphanumeric:
+                        revisionNumberingSequence = RevisionNumberingSequence.CreateAlphanumericSequence(doc, "Dynamo_Alphanumeric", new AlphanumericRevisionSettings());
+                        revisionNumberingSequenceId = revisionNumberingSequence.Id;
+                        break;
+                    case RevisionNumberType.Numeric:
+                        revisionNumberingSequence = RevisionNumberingSequence.CreateNumericSequence(doc, "Dynamo_Numeric", new NumericRevisionSettings());
+                        revisionNumberingSequenceId = revisionNumberingSequence.Id;
+                        break;
```

Comment:
no default branch?

Reply:
If not Alphanumeric or Numeric, all return invalidElementId. And revisionNumberingSequenceId was set to invalid element id when it initializes.